documentation/src/main/asciidoc/topics/code_examples/CustomMergePolicy.java

```diff
@@ -1,4 +1,5 @@
-ConfigurationBuilder dcc = new ConfigurationBuilder();
-dcc.clustering().partitionHandling()
-                    .whenSplit(PartitionHandling.ALLOW_READ_WRITES)
-                    .mergePolicy(new CustomMergePolicy());
+ConfigurationBuilder builder = new ConfigurationBuilder();
+builder.clustering().cacheMode(CacheMode.DIST_SYNC)
+       .partitionHandling()
+       .whenSplit(PartitionHandling.DENY_READ_WRITES)
+       .mergePolicy(new CustomMergePolicy());
```

Comment:
We're missing a JSON and YAML configuration that shows how to define a custom merge policy.